lib/Support/CMakeLists.txt

```diff
@@ -21,6 +21,7 @@ add_circt_library(CIRCTSupport
   PrettyPrinterHelpers.cpp
   ParsingUtils.cpp
   SymCache.cpp
+  Utils.cpp
```

Comment:
(note to self: fix InstanceGraph.cpp not being in the alphasort like the rest -- not for this PR)

Reply:
I actually had a commit that sorted the lines of this, but didn't land that. Good thinking.

Reply:
Sorted in 27968fcfe.